lib/foobara/util/constants.rb

```diff
@@ -0,0 +1,3 @@
+module Foobara
+  EMPTY_ARRAY = [].freeze
+end
```

Comment:
I think could rebase against foobara/util's main would get rid of these. Could also make a new branch and cherry pick your new changes. Or an interactive rebase in your branch to drop these commits.